assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -464,8 +464,12 @@ che.infra.kubernetes.client.http.connection_pool.keep_alive_min=5
 # If not set, every workspace will be created in a new project, where project name = workspace id
 che.infra.openshift.project=
 
-# Create routes with Transport Layer Security (TLS) enabled
-che.infra.openshift.tls_enabled=false
+# Creates Ingresses with Transport Layer Security (TLS) enabled
+# In OpenShift infrastructure, Routes will be TLS-enabled
+che.infra.kubernetes.tls_enabled=false
```

Comment:
Please move new Kubernetes properties above to Kubernetes section.

Reply:
done